src/wp-includes/widgets/class-wp-nav-menu-widget.php

```diff
@@ -114,15 +114,27 @@ public function update( $new_instance, $old_instance ) {
 	 */
 	public function form( $instance ) {
 		global $wp_customize;
-		$title = isset( $instance['title'] ) ? $instance['title'] : '';
+		$title    = isset( $instance['title'] ) ? $instance['title'] : '';
 		$nav_menu = isset( $instance['nav_menu'] ) ? $instance['nav_menu'] : '';
 
 		// Get menus
 		$menus = wp_get_nav_menus();
 
+		$empty_menus_style = $not_empty_menus_style = '';
+		if ( empty( $menus ) ) {
+			$empty_menus_style = ' style="display:none" ';
+		} else {
+			$not_empty_menus_style = ' style="display:none" ';
+		}
+
+		$nav_menu_style = '';
+		if ( ! $nav_menu ) {
+			$nav_menu_style = 'display: none;';
+		}
+
```

Comment:
This is far away from where it is used, and is more legible inline.
If it has to be separate, can it be a ternary?
`$nav_menu_style = $nav_menu ? '' : 'display:none';`

Reply:
This matches upstream, we can consider ternary use.
https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/widgets/class-wp-nav-menu-widget.php#L159

Reply:
Upstream is in favor of long format `if` for readability sake.

Reply:
Updated PR to follow